content/en-us/reference/engine/classes/Debris.yaml

```diff
@@ -3,64 +3,73 @@ type: class
 category: Utility
 memory_category: Instances
 summary: |
-  The Debris service allows the developer to schedule the removal of the object
-  without yielding any code, through the usage of the `Class.Debris:AddItem()`
+  The Debris service allows the developer to schedule guaranteed destruction of
+  the object without yielding, through the usage of the `Class.Debris:AddItem()`
   method.
```

Comment:
```suggestion
  Allows scheduling of guaranteed destruction of an object without yielding.
```

Reply:
IMHO summaries can be a bit shorter and not mention other methods within the class.

Reply:
I agree, I suppose it was included before just because there is only one useful api member, but it doesn't seem necessary to me to call out here.

Reply:
Removing the first `of`, it reads better I think
> Allows scheduling guaranteed destruction of an object without yielding.